test/e2e/atlas/kubernetes_config_generate_test.go

```diff
@@ -293,6 +300,9 @@ func TestProjectWithNonDefaultAlertConf(t *testing.T) {
 			strconv.Itoa(*newAlertConfig.Notifications[0].DelayMin),
 			fmt.Sprintf("--notificationSmsEnabled=%s", strconv.FormatBool(*newAlertConfig.Notifications[0].SMSEnabled)),
 			fmt.Sprintf("--notificationEmailEnabled=%s", strconv.FormatBool(*newAlertConfig.Notifications[0].EmailEnabled)),
+			fmt.Sprintf("--matcherFieldName=%s", *&newAlertConfig.Matchers[0].FieldName),
```

Comment:
same common comment, use the getters to avoid pointer logic

Reply:
I see no getter for matchers or notifications, you mean we should add them here?

Reply:
BTW matchers do not have pointers, so there is no need for getter or pointer logic

Reply:
I can't add getters on types that live on the operator code, I will create a ticket for that instead.

Reply:
CLOUDP-242585